add_new_employee_webhook_receiver.py

```diff
@@ -16,35 +17,44 @@ def add_employee_to_guide(event, context):
     try:
         # Fetch the Builder API key, the guide ID of the guide where the content
         # is published, and the custom list ID that the items are associated with
-        api_key, guide_id, employee_customlist_id, zenefits_app_key = fetch_ssm_params()
+        api_key, guide_and_list_ids, zenefits_app_key = fetch_ssm_params()
```

Comment:
Would it make sense to have `fetch_ssm_params()` return a dictionary of form `{"{ssm param}": value, ...}` rather than a tuple here?

Reply:
is a dictionary preferred? they both make sense to me

Reply:
Thinking about this more, if we make any changes here I think it should actually be a named tuple, not a dictionary. This would look something like:
```python
from collections import NamedTuple
...
def fetch_ssm_params():
    # ...
    SSMParams = NamedTuple('SSMParams', ['api_key', 'guide_and_list_ids', 'zenefits_app_key'])
    ...
    return SSMParams(api_key= , guide_and_list_ids= , ...)

# whenever we're referencing this:
ssm_params = fetch_ssm_params()

ssm_params.api_key ...
```
This might be overkill for our use case here though, so I'll leave it up to you on whether this is an improvement or not. I think at best it means simpler code changes if the return values of `fetch_ssm_params()` change. At worst, it's added complexity to something which might not change in the future. Generally though I think named tuples are a good tool to consider for situations like this.

Reply:
I'm indifferent about either way. There are few enough values returned now where I think it's manageable to keep as is. If more values are added in the future it for sure should be updated to a dict/named tuple

Reply:
Yeah, I think I'll leave it out on this one, as there are only three values. I added it for the google drive integration though, which has a lot more.